relnotes/nostart.migration.md

```diff
@@ -0,0 +1,9 @@
+### GetRange notification types started, suspended and resumed are removed
+
+Since the GetRange v1, the request is immediately ready to be suspended,
+and it also suspends and resumes instantly, without waiting for the nodes.
+This makes `started`, `suspended` and `resumed` notification types useless
+and they are now removed. The `started` notification should be replaced with
```

Comment:
"The `started` notification should be replaced with the code that follows the creation of the request"

This sentence doesn't make sense to me. If the started notification no longer exists, how can it's code be replaced with something else?

Reply:
I see I made mistake there, I'll reword:

"The code inside `started` notification should be placed after creation of the request."

Reply:
Ah, right, that makes more sense.